lib/usagereporter/daemon/usagereporter.go

```diff
@@ -132,6 +132,8 @@ func GetAnonymizedPrehogEvent(req *teletermv1.ReportUsageEventRequest) (*prehogv
 	}
 
 	// anonymized
+	// NOTE: for simplicity reasons the teleterm anonymization doesn't match the on-prem teleport anonymization.
+	// see https://github.com/gravitational/teleport/pull/35652#issuecomment-1865135970 for details
```

Comment:
Bumping the Rafał's comment https://github.com/gravitational/teleport/pull/35652#issuecomment-1865922431

@zmb3 do we need a confirmation from the product team?

Reply:
This is fine. The usage data we collect in connect is to help us build the product, not to bill customers.